libraries/AP_Camera/AP_Camera.cpp

```diff
@@ -404,6 +412,11 @@ void AP_Camera::log_picture()
         return;
     }
     if (!using_feedback_pin()) {
+        if(_relay_on == _feedback_polarity){
+            hal.scheduler->delay(_trigger_delay);
```

Comment:
This is absolutely unsafe to do in flight as the entire flight code will be stalled for the duration.

Reply:
Thank you for your quick response. Is there a safer way to delay the Camera feedback message?

Reply:
You would need to store a timestamp for when you recieved the feedback signal, then check next time you are updated if it has been that long. Can I ask why you want to delay the message though? The event you are trying to tag is happening when the feedback signal is driven, and that is the event one normally is trying to tag.

Reply:
Sorry for the late response. I want to delay the CAM_FEEDBACK by few milliseconds so that the picture taken and the geotags are in sync. Thank you for your response